src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -124,17 +130,18 @@ private static VersionInfo GetVersionInfo()
                 modelSignature: "HASHTRNS",
                 // verWrittenCur: 0x00010001, // Initial
                 // verWrittenCur: 0x00010002, // Invert hash key values, hash fix
-                verWrittenCur: 0x00010003, // Uses MurmurHash3_x86_32
-                verReadableCur: 0x00010003,
+                //verWrittenCur: 0x00010003, // Uses MurmurHash3_x86_32
+                verWrittenCur: 0x00010004, // Add Combine option
+                verReadableCur: 0x00010004,
                 verWeCanReadBack: 0x00010002,
                 loaderSignature: LoaderSignature,
                 loaderAssemblyName: typeof(HashingTransformer).Assembly.FullName);
         }
 
-        private readonly HashingEstimator.ColumnOptionsInternal[] _columns;
+        private readonly HashingEstimator.ColumnOptions[] _columns;
         private readonly VBuffer<ReadOnlyMemory<char>>[] _keyValues;
         private readonly VectorDataViewType[] _kvTypes;
-        private readonly bool _isVersion1;
+        private readonly bool _useOldHashing;
 
```

Comment:
suggestion: Is it possible to come up with a better name? The only one I can think of is "_useOnnxCompatibleHashing" which would invert the meaning of the current name. Please use a a more descriptive name if possible.

Reply:
I would like to keep it a variable that would have the value "false" by default, that's why I didn't want to change it to `_useOnnxCompatibleHashing`. Is this name that I changed it to acceptable?

---
In reply to: [424567216](https://github.com/dotnet/machinelearning/pull/4828#discussion_r424567216) [](ancestors = 424567216)